.github/workflows/label-checker.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
     check_for_matching_labels:
-      runs-on: ubuntu-22.04
+      runs-on: self-hosted
```

Comment:
This can run on default GHA runners as it only interacts with GitHub labels and not with the source code.

Reply:
done